.travis.yml

```
@@ -81,6 +81,8 @@ script:
   - |
     if [[ -v MAVEN_CHECKS ]]; then
       ./mvnw install -DskipTests -B -T C1 -P ci
+      PRESTO_VERSION=`mvn -Dexec.executable='echo' -Dexec.args='${project.version}' --non-recursive exec:exec -q`
```

Use mvn wrapper, reorder params:

```suggestion
      PRESTO_VERSION=`./mvnw --quiet --batch-mode --non-recursive  exec:exec -Dexec.executable='echo' -Dexec.args='${project.version}'`
```

---

Done.